app/presenters/special_users_presenter.rb

```diff
@@ -0,0 +1,26 @@
+#= Presenter for Setting / special users
+class SpecialUser
```

Comment:
The class name and file name should correspond to each other. In this case, since the class is a presenter for all special users, like the existing `Features` presenter which does something similar, I think `SpecialUsers` is the best name for the class (and so, `special_users.rb` for the file name).

Reply:
Thanks!
There was (is) a mess of concepts in my head: model name vs table name, presenters and "helper" wrapper class in presenters' folder. And not so good knowledge of tests environment and no complete understanding of how and what is loaded for tests.
Good to have a real life example to get some ground